Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -569,8 +569,8 @@ define([
             }
             that._credit = credit;
 
-            that._urlParts = urlTemplateToParts(that._url, tags);
-            that._pickFeaturesUrlParts = urlTemplateToParts(that._pickFeaturesUrl, pickFeaturesTags);
+            that._urlParts = urlTemplateToParts(that._url, tags); //eslint-disable-line no-use-before-define
```

Comment:
I don't think this or the next line is needed, I removed the comment and eslint didn't complete (which makes sense because `tags` is technically declared before this code is executed.  Are you seeing something difference?